packages/connect-common/files/coins.json

```diff
@@ -2253,25 +2294,6 @@
             },
             "url": "https://www.rinkeby.io"
         },
-        {
```

Comment:
looks like we did not update trezor-common with goerli. 

could you please, @karliatto have a look? also we were about to get rid of tGOR etc. and replace it with tETH instead. can we do it now, or are we still waiting for blockbooks?

Reply:
Can we coordinate this for the March release? It's the next release of FW.

Reply:
> But it tGOR is still there in https://github.com/trezor/trezor-common/blob/master/defs/coins_details.json

don't look at that file .)

I was under the impression that we decided to _keep_ `tGOR` and friends (bc it makes it easier to see on Trezor on which testnet you are), no?

current master: https://github.com/trezor/trezor-common/blob/master/defs/duplicity_overrides.json#L18-L19

Reply:
Intention is to sync with upstream and avoid special handling.